src/VS.Web.CG.Mvc/Identity/IdentityGeneratorTemplateModelBuilder.cs

```diff
@@ -261,8 +261,22 @@ private bool DetermineSupportFileLocation(out string supportFileLocation, out st
 
             if (!string.IsNullOrEmpty(_commandlineModel.Layout))
             {
-                supportFileLocation = Path.GetDirectoryName(_commandlineModel.Layout);
-                layoutFile = _commandlineModel.Layout;
+                if (_commandlineModel.Layout.StartsWith("~"))
+                {
+                    layoutFile = _commandlineModel.Layout.Substring(1);
+                }
+                else
+                {
+                    layoutFile = _commandlineModel.Layout;
+                }
+
+                if (layoutFile[0] == '\\' || layoutFile[0] == '/')
```

Comment:
This already handles `~/` and `~\` cases, should this support crazy input like `\\\abc\layout` or `\/abc\layout`

Reply:
Yes, it probably should. The path is assumed to always be relative to the project root, so all lead slashes should be removed. Fixing...